### PR TITLE
Added the option to remember what filter you used last

### PR DESCRIPTION
**Pull request by nemotime**
and load it back whenever you resume using the app. Including when you close it.

As per User Story #9

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt [hunk 1]
```diff
@@ -206,6 +205,7 @@ class MainActivity : SimpleActivity(), PreviewListener, PhotoProcessor.MediaSave
         view_holder.addView(mFocusRectView)
 
         mIsInPhotoMode = true
+        //set old filter
```
Comment (Earthii) [inline app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt]: useless comment

File: app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt [hunk 2]
```diff
@@ -15,6 +15,7 @@ import android.view.ScaleGestureDetector
 import android.view.SurfaceHolder
 import android.view.SurfaceView
 import android.view.ViewGroup
+import android.widget.Toast
```
Comment (Earthii) [inline app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt]: dont think you need this anymore

File: app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt [hunk 3]
```diff
@@ -153,6 +154,7 @@ class Preview : ViewGroup, SurfaceHolder.Callback, MediaScannerConnection.OnScan
             return false
 
         mParameters = mCamera!!.parameters
+        Log.d("TAG", mParameters!!.colorEffect)
```
Comment (Earthii) [inline app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt]: Take this out

File: app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt (hunk 2 above)
Comment (nemotime) [reply app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt]: I still need this, I did use it for my testing but the applications sends error messages as toasts

File: app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt (hunk 1 above)
Comment (nemotime) [reply app/src/main/kotlin/com/simplemobiletools/camera/activities/MainActivity.kt]: removed

File: app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt (hunk 3 above)
Comment (nemotime) [reply app/src/main/kotlin/com/simplemobiletools/camera/views/Preview.kt]: removed

**Comment by Earthii** [comment]
Good job :)

**Pull request merged**